cmd/juju/caas/add.go

```diff
@@ -44,10 +46,17 @@ Adds a k8s endpoint and credential to Juju.`[1:]
 
 var usageAddCAASDetails = `
 Creates a user-defined cloud and populate the selected controller with the k8s
-cloud details.
+cloud details. Speficify non default kubeconfig file location using $KUBECONFIG
+environment variable or pipe in file content from stdin. The config file
+can contain definitions for different k8s clusters, use --context-name to pick
+which one to use.
 
 Examples:
-    juju add-k8s myk8scloud`
+	juju add-k8s myk8scloud
+	juju add-k8s myk8scloud
+	KUBECONFIG=path-to-kubuconfig-file juju add-k8s myk8scloud [--context-name=my_context_name]
```

Comment:
We give full examples here rather than options, so remove the [] and make a new example if needed